biorbd_optim/optimal_control_program.py

```diff
@@ -515,27 +524,38 @@ def solve(self, solver="ipopt", show_online_optim=False, options_ipopt={}):
         }
 
         # Solve the problem
-        return solver.call(arg)
+        out = solver.call(arg)
+
+        if save_iterations:
+            with open(file_path, "rb") as file:
+                iterations = pickle.load(file)
+                out = out, iterations
+                os.remove(file_path)
+        return out
 
-    def save(self, sol, file_path):
+    def save(self, sol, file_path, iterations=[]):
         _, ext = os.path.splitext(file_path)
         if ext == "":
             file_path = file_path + ".bo"
         elif ext != ".bo":
             raise RuntimeError(f"Incorrect extension({ext}), it should be (.bo) or (.bob) if you use save_get_data.")
+        dict = {"ocp_initilializer": self.original_values, "sol": sol, "versions": self.version}
+        if iterations != []:
```

Comment:
I think I prefer None

Reply:
Does it even make sense to allow for this? I mean this fundamentally different no?